chibuezeayogu/React-Redux
# #161532400 Implement CRUD for authors

**What does the PR do?**
- This PR Implements  `Authors CRUD`

**Description of Task to be completed?**
-  implement `ADD`, `VIEW`, `EDIT` and `DELETE` authors
 
**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Redux.git)
-  cd into `React-Redux` and checkout to this branch `ft-implement-author-add-view-edit-delete-161532400`
- run the command `yarn dev` to start the application locally.
- run the command `yarn test` to ensure all tests are passing.

**What are the relevant pivotal tracker stories?**
- [#161532400](https://www.pivotaltracker.com/story/show/161532400)

**Screenshots**
- N/A
